xcmailr-webapp/src/main/java/controllers/AdminHandler.java

```diff
@@ -167,20 +165,21 @@ public Result pagedMTX(Context context, @Param("p") int page)
      *            the time in days (all before will be deleted)
      * @return mail-transactions overview page
      */
-    public Result deleteMTXProcess(@PathParam("time") Integer time, Context context)
+    public Result deleteMTXProcess(@PathParam("time") Optional<Integer> time, Context context)
```

Comment:
Time is parsed as Integer and defaults to `null` in case it could not be parsed successfully. For this reason and since implementation does not require the parameter to be given, I made it optional.